src/core/common/Sidebar/MobileMenu.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from "react";
+import { connect } from 'react-redux';
+import { setSidebarVisibility } from 'react-admin';
+
+import MenuItems from "./MenuItems";
+
+class MobileMenu extends Component {
+
+    onMobileMenuClick = () => {
+        this.props.setSidebarVisibility(true);
+    };
+
+    render() {
+        const { classes, menuItems, currentList } = this.props;
+        return (
+            <div className={classes.mobileSidebar}>
+                <MenuItems classes={classes} menuItems={menuItems} currentList={currentList} onMenuClick={() => this.onMobileMenuClick()} />
+            </div>
+        );
+    }
+};
+
+const mapDispatchToProps = dispatch => {
```

Comment:
use the mush shorter form:

```js
const mapDispatchToProps = { setSidebarVisibility };
```

https://react-redux.js.org/using-react-redux/connect-mapdispatch#defining-mapdispatchtoprops-as-an-object